Revise the positionConfig contents (#650)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/positionConfig/of.md

---
---
##### shortDescription

> The target element against which the overlay element should be positioned.

Reviewer: Do you mean:
The target element where an overlay element should be positioned.

Author: ```suggestion
The target element relative to which the overlay element should be positioned.
```

Reviewer: Do you mean:
The target element over which the overlay element is positioned.

                DxPosition
            },
            data() {
                return {
                    target: window
                }
            }
        }
        </script>
    
    ##### React
    
        <!-- tab: App.js -->
        import React from 'react';
    
        import 'devextreme/dist/css/dx.common.css';
        import 'devextreme/dist/css/dx.light.css';
    
        import Popup, {
            Position
        } from 'devextreme-react/popup';
    
        class App extends React.Component {
            constructor(props) {
                super(props);
                this.target = window;
            }
            render() {
                return (
                    <Popup>
                        <Position
                            of={this.target}
                        />
                    </Popup>
                );
            }
        }
        export default App;
    
    ---

- [dxEvent](/api-reference/50%20Common/Object%20Structures/dxEvent '/Documentation/ApiReference/Common/Object_Structures/dxEvent/') (or <a href="http://api.jquery.com/category/events/event-object/" target="_blank">jQuery.Event</a> if jQuery is used)

>  The overlay element is positioned at the **event.pageX** and **event.pageY** coordinates. In the following example, the [Popover](/api-reference/10%20UI%20Widgets/dxPopover '/Documentation/ApiReference/UI_Widgets/dxPopover/') widget is positioned at a clicked point on the target.

Reviewer: ```suggestion
 The overlay element is positioned at the **event.pageX** and **event.pageY** coordinates. In the following example, the [Popover](/api-reference/10%20UI%20Widgets/dxPopover '/Documentation/ApiReference/UI_Widgets/dxPopover/') widget is positioned at the point where the user has clicked.
```
